.planning/milestones/v1-REQUIREMENTS.md

```
@@ -0,0 +1,160 @@
+
+This is the archived requirements specification for v1.
+For current requirements, see `.planning/REQUIREMENTS.md` (created for next milestone).
+
+---
+
+# Requirements: Netro Plugin Refactoring
+
+**Defined:** 2026-02-01
+**Core Value:** Maintain reliable, maintainable Indigo plugin with clean, testable code
+
+## v1 Requirements
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)